android/src/main/java/io/customer/reactnative/sdk/messagingpush/RNCIOPushMessaging.kt

```diff
@@ -66,6 +76,37 @@ class RNCIOPushMessaging(
         }
     }
 
+    /**
+     * Handles push notification received. This is helpful in processing push notifications
+     * received outside the CIO SDK.
+     *
+     * @param message push payload received from FCM.
+     * @param handleNotificationTrigger indicating if the local notification should be triggered.
+     */
+    @ReactMethod
+    fun handleMessage(message: ReadableMap?, handleNotificationTrigger: Boolean, promise: Promise) {
+        try {
+            if (message == null) {
+                promise.reject(IllegalArgumentException("Remote message cannot be null"))
+                return
+            }
+
+            // Generate destination string, see docs on receiver method for more details
+            val destination = message.getString("to")?.takeIfNotBlank()
+                ?: CustomerIO.instanceOrNull(reactContext)?.diGraph?.sitePreferenceRepository?.getIdentifier()
+                ?: UUID.randomUUID().toString()
```

Comment:
We probably don't need this 
```
CustomerIO.instanceOrNull(reactContext)?.diGraph?.sitePreferenceRepository?.getIdentifier()
```

as it won't be mapped to GCM sender ID, just making is null safe would be suffice

Reply:
Yeah, it would. Updated code to remove identifier part.